### ISSUE TITLE
Scss files doesn't works

### ISSUE THREAD

**Issue by josefosaurus** [livestyle/issues#26]
Hi, I'm trying to  use scss files in this extension but doesn't show in the dropdown selector of files in the chrome extension.

**Comment by sergeche** [comment]
Did you set syntax to SCSS? Can you provide a screenshots of your editor and Chrome with LiveStyle popup opened?

**Comment by josefosaurus** [comment]
In my laptop at house the less files cam be loaded but in my job none of
both load's less or scss none of those two, the css files are recognized

**Comment by runofthemill** [comment]
@j-avila is your editor is identifying the files as 'SCSS' and not 'Sass'? in Sublime Text 3 try the package [Syntax Highlighting for Sass](https://packagecontrol.io/packages/Syntax%20Highlighting%20for%20Sass) which has an option for both Sass and SCSS

**Comment by sergeche** [comment]
This bug is still relevant?

**Comment by robvr01** [comment]
I'm also having this problem. Expect it's due to using a lot of @import rules?

<img width="957" alt="screen shot 2016-01-01 at 15 07 04" src="https://cloud.githubusercontent.com/assets/10153399/12071250/8c4115aa-b099-11e5-91ff-dc6c885b2788.png">

**Comment by mastergizmo** [comment]
@runofthemill -s hint did the trick fo me.

Install the Package [Syntax Highlighting for Sass](https://packagecontrol.io/packages/Syntax%20Highlighting%20for%20Sass)
open a ".scss" File and set set the mapping to "Open all with current extension as..." -> "Syntax highlighting for Sass" -> "SCSS"

After that, the ".scss" Files should show up in the Chrome Extension.

![scss](https://cloud.githubusercontent.com/assets/961516/12071449/c6fdb0f4-b0ad-11e5-8ca3-72a483661020.png)

**Comment by sergeche** [comment]
@robvr01 no, lots of imports shouldn’t be a problem to display file in list (although it can slow things down, but I will optimize it). As @mastergizmo noted, they should have proper syntax and it must `scss` (SASS is not supported right now)

**Comment by robvr01** [comment]
@mastergizmo This worked! Thanks so much for your help :) 
@sergeche This is a fantastic tool, now I can use it for scss it makes editing css on live sites really good.

**Comment by j-avila** [comment]
Hola,

Acabo de firmar la petición de "@OACNUDH @UNHumanRights Visita de
@mbachelet a Venezuela para constatar la violación de DDHH" y me encantaría
que me ayudaras sumando tu firma.

Nuestro objetivo es llegar a 75,000 firmas y necesitamos más apoyo. Para
obtener más información y firmar la petición puedes abrir el siguiente link:

http://chng.it/LsfcF57Kvy

¡Gracias!
jose
